.changeset/quiet-eels-rest.md

```diff
@@ -0,0 +1,5 @@
+---
+"create-cloudflare": patch
+---
+
+feat: add experimental Next.js, with Workers assets, template
```

Comment:
```suggestion
chore: add experimental Next.js, with Workers assets, template
```

Reply:
this is a feature not a fix. I'm asking for an exemption from the feature freeze